apps/guide/src/content/05-additional-info/02-collections.mdx

```diff
@@ -28,8 +28,8 @@ Many of the methods on _`Collection`_ correspond to their namesake in _`Array`_.
 
 ```js
 // Assume we have an array of users and a collection of the same users.
-array.find((u) => u.discriminator === '1000');
-collection.find((u) => u.discriminator === '1000');
+array.find((user) => user.bot === true);
+collection.find((user) => user.bot === true);
```

Comment:
```suggestion
array.find((user) => user.bot);
collection.find((user) => user.bot);
```

Reply:
Idk if looking for bots is a good example
user.username.includes("cat") looks cooler

Reply:
This code chunk was originally to filter for users with the discriminator of `#1000`, so I think @jaw0r3k's suggestion would be better suited for this use case..

Still waiting to see if this PR will be considered necessary since there is a discussion about `.tag` being "un-deprecated".

Reply:
Disciminators wont be used by regular users
So this example should be changed
But leave tag code as bots still uses it